jupyter-extension/src/plugins/mount/components/Pipeline/__tests__/Pipeline.test.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {render} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import * as requestAPI from '../../../../../handler';
+import {mockedRequestAPI} from 'utils/testUtils';
+import Pipeline from '../Pipeline';
+jest.mock('../../../../../handler');
+
+describe('PPS screen', () => {
+  let setShowPipeline = jest.fn();
+  const mockRequestAPI = requestAPI as jest.Mocked<typeof requestAPI>;
+
+  beforeEach(() => {
+    setShowPipeline = jest.fn();
+    mockRequestAPI.requestAPI.mockImplementation(mockedRequestAPI({}));
+
+    // IntersectionObserver isn't available in test environment
+    const mockIntersectionObserver = jest.fn();
```

Comment:
You can remove these lines related to `mockIntersectionObserver`. This is used for tests that determine whether an HTML element is visible on the screen (different from when it's rendered but hidden behind another element).